x-pack/packetbeat/npcap/installer/README

```diff
@@ -0,0 +1,7 @@
+This directory is pinned to allow placement of the Npcap OEM installer
+during x-pack Packetbeat builds.
+
+Only one exe file may be placed here and this should be the most recent
+available Npcap installer. As part of the distribution a place-holder file
+is put here to allow compilation. This file must be removed when a real
```

Comment:
Requiring the place-holder to be removed leaves the repo is a modified state. Is there a way to implement this and keep the repo in an unmodified state?

Reply:
I thought about that and there is no ideal approach. The background here is that embedding is a compile time operation, so at least one file must match the `go:embed` directive glob. The choice I made for an invariant is that there is always one match. This was to make it completely clear when a binary is embedded. The intention is then to have a `go generate` tool that will do the work of obtaining the artifact from the bucket during build and replacing the placeholder. The alternative is to have logic that allows 1 or 2 matches and picks the highest. I don't like the relaxed invariant as much, but I'll investigate the implications.

Reply:
Done